sdk/search/tests.yml

```diff
@@ -14,7 +14,7 @@ extends:
     CloudConfig:
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-        Location: 'eastus2euap'
+        Location: 'westcentralus'
```

Comment:
Have the changes reached westcentralus?

Reply:
Yes, all the changes are available in these regions

![image](https://github.com/Azure/azure-sdk-for-net/assets/45216704/5cdeae09-c4b1-42a6-88e5-00a0b0f83c7c)

Reply:
You may need to move this back to EUAP when you start with GA swagger.

Reply:
Some of my existing test cases where we use [SEARCH_COGNITIVE_KEY](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/test-resources.json#L203C10-L203C30) test resource was failing with [this](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3136875&view=logs&j=70ccd553-3427-5ec1-2920-06cca04de773&t=ac2c2ae1-f45f-5f65-4299-7a0ae8f99294&l=544) error:

> Error Message:Azure.RequestFailedException : Provided key is not a valid CognitiveServices type key for the region of your search service.
Status: 400 (Bad Request

Updating the `westcentralus` region works well for all the resources.